### ISSUE TITLE
Upgrade nsp

### ISSUE THREAD

**Issue by chenba** [mozilla-services/screenshots#4324]
This is a follow-up to #4280.  The newer version of `nsp` is not ignoring the advisories we ask it to in `.nsprc`.  A configuration change is probably needed.

**Comment by chenba** [comment]
Actually, let's wait on https://github.com/nodesecurity/nsp/issues/207 for this.

**Comment by chenba** [comment]
Now that npm has acquired NSP: "This service remains operational for current users in its current state. No new features or fixes will be implemented."  The nsp repo has been archived and that above link to the issue returns a 404. :disappointed:

**Comment by pdehaan** [comment]
Actually, it looks like [the Node Security Platform (nsp) service is shutting down 9/30](https://blog.npmjs.org/post/175511531085/the-node-security-platform-service-is-shutting).
They recommend switching to [npm@6's <kbd>$ npm audit</kbd>](https://docs.npmjs.com/getting-started/running-a-security-audit) as a replacement.

**Comment by 6a68** [comment]
Still no update on a replacement for `.nsprc` in npm 6

**Comment by pdehaan** [comment]
I'm not a huge fan of <kbd>npm audit</kbd> yet, as it seems like a poor clone of the **nsp** service. Plus, forcing everybody onto npm@6 seems poor.

I know some other projects are using https://snyk.io/ to manage vuln detection.

**Comment by wresuolc** [comment]
closing in favor of #4704
